### PR TITLE
Feat/cli dump

### PR DESCRIPTION
**Pull request by gmasgras**
Implements https://github.com/ipfs-shipyard/go-dumpotron/issues/2

### CODE REVIEW

**Comment by gmasgras** [comment]
@olizilla please to have review ?

**Review by coryschwartz — APPROVED** [review]

**Review by olizilla — APPROVED** [review]
Ran it locally. IT DUMPS!

File: README.md [hunk 1]
```diff
@@ -23,16 +23,31 @@ Docker:
...
+PPROF_AUTH_PASS=THE_ADMIN_HTTPASSWD ./go-dumpotron gateway-bank1-ewr1.dwebops.net
+```
+
+Docker:
+```
+mkdir /tmp/dumps
+docker run --rm --net=host --name=go-dumpotron --env-file=.dockerenv -v /tmp/dumps:/tmp go-dumpotron gateway-bank1-ewr1.dwebops.net
+```
+
+### Webhook Test:
```
Comment (olizilla) [inline README.md]:
```suggestion
### Webhook Test
```

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The following ENVs need to be set:
 - IPFS_CLUSTER_AUTH
 - GITHUB_TOKEN
 
-Build:
+### Build:
```
Comment (olizilla) [inline README.md]:
```suggestion
### Build
```

**Comment by olizilla** [comment]
Minor, but it'd be nice if the archive was moved to the cwd when doing a local dump. I'm unlikely to remember where the temp folder is if i need to find it in the future, and I probably wont ever clean it up, nor will OSX, if it's hidden away in a temp.

**Comment by gmasgras** [comment]
> Minor, but it'd be nice if the archive was moved to the cwd when doing a local dump. I'm unlikely to remember where the temp folder is if i need to find it in the future, and I probably wont ever clean it up, nor will OSX, if it's hidden away in a temp.

:+1: Will address in a PR

**Pull request merged**